plugin/advisor/catalog/walk_through.go

```diff
@@ -1045,63 +1145,67 @@ func (t *tableState) createIndex(name string, keyList []string, unique bool, tp
 		}
 	}
 
-	index := &indexState{
+	index := &IndexState{
+		complete:       true,
```

Comment:
Do we need to differentiate the state to be in-completed or completed? When we do the walk-through, it has to be either completed or in-completed.